mosaic-site/src/components/mdx/h2/Heading2.tsx

```diff
@@ -1,6 +1,13 @@
-import { ReactNode } from "react";
+import { FC, ReactNode } from "react";
 import styles from "./Heading2.module.css";
 
-export const Heading2 = ({ children }: { children: ReactNode }) => (
-  <h2 className={styles.heading2}>{children}</h2>
+type Heading2Props = {
+  children: ReactNode;
+  id?: string;
+};
+
+export const Heading2: FC<Heading2Props> = ({ children, ...rest }) => (
```

Comment:
Could you have done:

```
import { H2 } from '@jpmorganchase/mosaic-components';
const Heading2 = props => <H2 className={styles.heading2} {...props} />;
```

The other thing to note is that there is an `H2` component and and `h2` element (as this is part of markdown)

So you *may* find a discrepancy now that you have only changed the style of the component

Reply:
I've tried this approach initially but the problem was that it still pulls all the mosaic styles that I would then have to manually override so it seemed like the cleanest solution to me.
I've also added some base css styles that should cover for the `H2`/`h2` discrepancies